tests/unittest_manager.py

```diff
@@ -107,6 +108,11 @@ def test_ast_from_namespace_pkgutil(self) -> None:
     def test_ast_from_namespace_pkg_resources(self) -> None:
         self._test_ast_from_old_namespace_package_protocol("pkg_resources")
 
+    def test_identify_old_namespace_package_protocol(self) -> None:
+        self.assertTrue(
+            util.is_namespace("tests.testdata.python3.data.path_pkg_resources_1")
```

Comment:
hah, pre-commit just straight up deleted one of my imports to set up this test case, hang on